draft-ietf-ace-edhoc-oscore-profile.md

```diff
@@ -515,23 +508,17 @@ That is, C sends a POST request to the /authz-info endpoint at RS, with the requ
 
 The communication with the /authz-info endpoint does not have to be protected, except for the update of access rights case described below.
 
-In case of initial access token provisioning to RS for this Client, or in other cases without a valid EDHOC session, the uploading of the access token is followed by the execution of the EDHOC protocol (or combined using EAD as described in {{edhoc-exec}}) and by the derivation of an OSCORE Security Context, as detailed later in this section.
-
-In the following, we outline some alternative processing, which occur when the provisioned access token or the established OSCORE Security Context for various reasons is no longer available or sufficient. In the cases below, it is assumed that the EDHOC session is still valid, otherwise the processing essentially falls back to the case discussed above.
-
-1. C may be required to re-POST the access token, since RS may have deleted a stored access token (and associated OSCORE Security Context) at any time, for example due to all storage space being consumed. This situation can be detected by C when it receives a 4.01 (Unauthorized) response from RS, especially as an "AS Request Creation Hints" message (see {{Section 5.3 of RFC9200}}.
+The Client provisioning an initial access token to the RS is followed by the execution of the EDHOC protocol (or combined using EAD as described in {{edhoc-exec}}) and by the derivation of an OSCORE Security Context, as detailed later in this section.
 
-2. C may be posting the first access token in a new series, e.g., because the old access token expired and thus its series terminated.
+The same procedure applies in other cases without a valid OSCORE Security Context shared between the C and RS, for example:
```

Comment:
s/between the C and RS/between C and RS

Not sure if it's worth mentioning: for the cases listed below, this same procedure would apply irrespective of having a valid EDHOC session.

Reply:
I updated the text, but omitted to point out this specifically, since this is a question you more likely would ask if you had the mindset of the our previous discussion rather than if you read the draft without this background.